python-sdk-example/tests/integration/resources_test.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import print_function
+
+import time
+import unittest
+import os
+import random
+import requests
+import signadot_sdk
+import string
+from signadot_sdk.rest import ApiException
+
+def get_random_string(length):
+    return ''.join(random.choice(string.ascii_lowercase + string.digits) for i in range(length))
+
+# This test creates a Sandbox with below specification:
```

Comment:
Can you add a bit more about the pre-requisites here? i.e. hotrod must be installed, and the resource plugin must be installed.